.Rbuildignore

```diff
@@ -20,3 +20,4 @@
 ^cran-comments\.md$
 ^docs$
 ^_pkgdown\.yaml$
+^vignettes/compass-directions.Rmd
```

Comment:
Hmm, maybe I should have rebased after all.  But when I tried rebasing, every single file had changed -- perhaps because of windows vs unix vs mac line endings?

Reply:
Not sure. I moved over to ubuntu half way through working on this and haven't had any trouble.